ush/python/pygfs/ufswm/gfs.py

```diff
@@ -52,16 +52,22 @@ def build_ufs_model(self) -> None:
         # Get model dates relevant for this run
         self._get_date_info()
 
-        logger.debug(f"UFS-weather-model configuration:\n{pformat(self.ufs_model)}")
+        logger.debug(
+            f"UFS-weather-model configuration:\n{pformat(self.ufs_model)}")
 
     @logit(logger)
     def _get_fix_info(self) -> None:
         self.ufs_model.fix = AttrDict()
-        self.ufs_model.fix.FIX_aer = os.path.join(self.ufs_model.HOMEufs, 'fix', 'aer')
-        self.ufs_model.fix.FIX_am = os.path.join(self.ufs_model.HOMEufs, 'fix', 'am')
-        self.ufs_model.fix.FIX_lut = os.path.join(self.ufs_model.HOMEufs, 'fix', 'lut')
-        self.ufs_model.fix.FIX_orog = os.path.join(self.ufs_model.HOMEufs, 'fix', 'orog')
-        self.ufs_model.fix.FIX_ugwd = os.path.join(self.ufs_model.HOMEufs, 'fix', 'ugwd')
+        self.ufs_model.fix.FIX_aer = os.path.join(
+            self.ufs_model.HOMEufs, 'fix', 'aer')
+        self.ufs_model.fix.FIX_am = os.path.join(
+            self.ufs_model.HOMEufs, 'fix', 'am')
+        self.ufs_model.fix.FIX_lut = os.path.join(
+            self.ufs_model.HOMEufs, 'fix', 'lut')
+        self.ufs_model.fix.FIX_orog = os.path.join(
+            self.ufs_model.HOMEufs, 'fix', 'orog')
+        self.ufs_model.fix.FIX_ugwd = os.path.join(
+            self.ufs_model.HOMEufs, 'fix', 'ugwd')
```

Comment:
same comment as above.
The line-length set in this project is 120 characters.  And I am certain this is well below that mark.
So, why this change?